### ISSUE TITLE
`waypoint-odr` image does not have git

### ISSUE THREAD

**Issue by arbourd** [hashicorp/waypoint#3862]
**Describe the bug**

After deploying Waypoint to a Kubernetes cluster with Helm (manually and not with the CLI), we started to experience:

```
error during job execution: job_id=01GCS6E7KYEQ9APM98BQ5DM4H7 job_op=*gen.Job_Poll err="exec: "git": executable file not found in $PATH"
```

It seems that the [waypoint-odr](https://hub.docker.com/r/hashicorp/waypoint-odr) does not have git installed. If that is the case, then why is the **Job Poll** operation being handled on an ODR and not the main runner?

**Steps to Reproduce**

1. Install on Kubernetes with Helm (default values to ensure `waypoint-odr` is being used)
2. Configure Waypoint to use `git` and job polling on any repository
3. Tasks will fail trying to clone archive

**Expected behavior**

`git clone` operation succeeds.

**Waypoint Platform Versions**

Additional version and platform information to help triage the issue if
applicable:

* Waypoint CLI Version:
  ```
  CLI: v0.9.1 (9562f453c)
  Server: v0.9.1
  ```
* Waypoint Server Platform and Version: `kubernetes`)
* Waypoint Plugin: n/a

**Additional context** 

Switching the image to `waypoint` resolves the error.

**Comment by arbourd** [comment]
Seems resolved with `0.10.0` but if it returns I will reopen.

**Comment by arbourd** [comment]
Seems resolved with `0.10.0` but if it returns I will reopen.
